lib/Cake/Utility/String.php

```diff
@@ -20,673 +20,7 @@
  * String handling methods.
  *
  * @package       Cake.Utility
+ * @deprecated 3.0.0 Deprecated since version 2.7, use CakeText class instead.
```

Comment:
Most of it will most likely have to go into the migration guide and documentation.